src/main/java/com/google/sps/utility/ApiKeyProvider.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.utility;
+
+import com.google.appengine.repackaged.com.google.gson.Gson;
+import com.google.appengine.repackaged.com.google.gson.reflect.TypeToken;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+
+public final class ApiKeyProvider {
+  public static String get(String name) throws IOException {
```

Comment:
Nit: Public classes and methods require [Javadoc](https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting)

Reply:
Thanks for pointing it out!

Reply:
`RESOLVED`